stories/install.fmf

```diff
@@ -6,6 +6,23 @@ link:
 
 /minimal:
     summary: Minimal tmt installation with core features only
+    description: |
+        The minimal package provides basic functionality which
+        covers mainly these areas:
+
+        * explore and lint existing tests, plans and stories
+        * create new tests, plans and stories from templates
+        * execute tests on the :ref:`/plugins/provision/local`
+          host or :ref:`/plugins/provision/connect` to a remote
+          guest using ssh
+        * prepare guest using the
+          :ref:`/plugins/prepare/ansible`,
```

Comment:
How should we document that ansible is possible only if weak dependencies are also installed?

Reply:
@psss do you want to address this?

Reply:
Let's explicitly mention it. Added in 09cbc5c0.